### PR TITLE
Enable suport for JIT in PostgreSQL >= 11

### PR DESCRIPTION
**Pull request by biertie**
Dear, 

With this merge request JIT can be enabled on PostgreSQL 11.   
As per example from other packages, it is possible to enable JIT and to change the default package name. 

The change has been tested on RHEL only.

### CODE REVIEW

File: manifests/server/config.pp [hunk 1]
```diff
@@ -130,6 +131,15 @@
     }
   }
 
+  if $jit_enable {
```
Comment (npwalker) [inline manifests/server/config.pp]: What happens if you set jit_enable = true and then set it to false afterwards?

Comment (biertie) [reply manifests/server/config.pp]: Good point. I will rework it so it becomes a bool.

**Comment by lionce** [comment]
Hello @biertie ,

This seems to be a great feature. Did you have time to update your PR accordingly with @npwalker's suggestion? Would you please add unit tests for your additions, to prevent the future issues. Also please update README file explaining this new feature. Please let us know if you need any help on doing this. 
Thank you!

Cheers!

**Comment by biertie** [comment]
@lionce I am planning to work on it further somewhere next month. At the moment I'm to busy with other issues :-)

**Comment by florindragos** [comment]
Hi @biertie 
Is this still work in progress? There are some problems on non-debian systems: `Evaluation Error: Cannot reassign variable '$pgsql_llvmjit_package_name'` :)

**Comment by sheenaajay** [comment]
@biertie PR Looks good. Could you please add unit tests to verify the change and also suggestions from @npwalker.Ran the Adhoc test suites and it's all green.

**Comment by michaeltlombardi** [comment]
Since this hasn't seen any activity in about four months, I'm going to go ahead and close this pull request for the time being. Please re-open this pull request once the next actions are addressed, new information is available, or you have a question related to this pull request. We've become aware of difficulties re-opening pull requests, in the event you cannot please mention `puppetlabs/modules`  with an @ sign in front and we'll re-open this pull request.

Closing the pull request doesn't mean we don't consider this change valuable, just that there are things that need to be addressed before it can be merged. If you have any questions or concerns, please don't hesitate to ping us in slack.puppet.com.

**Pull request closed without merging**